### DISCUSSION TITLE
删减功能

### DISCUSSION THREAD

**Discussion by TTsXDU** [wh201906/SerialTest#53] [Q&A]
博主你好，我想针对我的项目对这个上位机的功能进行删减，只保留蓝牙通信，数据显示和绘图的功能，使上位机的界面更加简洁。请问我需要对哪些代码文件进行删减呢，烦请博主指点一下。

**Comment by wh201906** [comment]
你可以在对应的.ui文件里面找到需要隐藏的控件的名字，然后在程序初始化的时候调用`xx.hide()`即可

**Reply by TTsXDU** [reply]
好的，我去试试

**Reply by TTsXDU** [reply]
呃，博主你好，这个xx.hide()函数是在哪里添加呢，在.cpp文件中吗？程序初始化是怎么进行的呢？我不太懂，恳请博主详细指教下QAQ

**Reply by wh201906** [reply]
是在.cpp文件当中。具体添加到哪个位置取决于你要隐藏哪些东西，比如你想隐藏控制和文件收发面板，那就要在`MainWindow::MainWindow()`当中调用对应的`hide()`。如果是需要隐藏蓝牙以外的连接，就需要在`DeviceTab::DeviceTab()`当中隐藏无关的组件。

**Reply by TTsXDU** [reply]
好的，大概明白了，我去试试
